### ISSUE TITLE
[Accessibility] healthcare.gov logo's svg elements are being read out by VoiceOver

### ISSUE THREAD

**Issue by SparrowSpit** [CMSgov/design-system#2606]
**Describe the bug**
[Priority 3] The path and polygon elements within the svg element are being read out by VoiceOver. They should be hidden.

**To Reproduce**
1. Go to the [Healthcare.gov Header](https://design.cms.gov/storybook/iframe.html?globals=theme:healthcare&id=healthcare-header--minimal-header&viewMode=story) and open VoiceOver (+ Safari)
2. Go to the Images panel within the rotor in VoiceOver
3. Several images are listed
4. You can then select one of those images and you will be taken to the letters of the logo.

**Expected behavior**
Those svg elements are not useful to a screen reader user and should be hidden from the accessibility tree.

**Desktop (please complete the following information):**
 - VoiceOver and Safari Version 16.5.2 (18615.2.9.11.10)

Edited to note Priority level.

**Comment by pwolfert** [comment]
Fixed by https://github.com/CMSgov/design-system/pull/2726
